dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/readerOMC.py

```diff
@@ -1039,6 +1040,14 @@ def read_06inz_c_file(self):
                     (list_body, depend) = replace_dynamic_indexing(list_body)
 
                     for line in list_body:
+                        #Sometime a complex init ends up in 06...
+                        #data->modelData->realVarsData[...].attribute.start = data->simulationInfo->realParameter[..];
+                        #data->localData[0]->realVars[...]  = data->modelData->realVarsData[...].attribute.start;
+                        if ptrn_assign_var_complex.search(line) is not None:
+                            match = re.search(ptrn_assign_var_complex, line)
+                            var = str(match.group('varName'))
```

Comment:
The naming seems a bit confusing : the python variable named `var` is filled with the data extracted from the regex group named `"varName"` whereas in the same regex there is another group named `"var"` which seems to be unused. Is my understanding correct?